### PR TITLE
flyToBounds tests added

### PR DESCRIPTION
**Pull request by shreya024**
Implemented tests for flyToBounds method of Map(#8145)

### CODE REVIEW

File: spec/suites/map/MapSpec.js [hunk 1]
```diff
@@ -2499,7 +2627,118 @@ describe('Map', () => {
...
+				[-1, -1],
+				[1, 1],
+			]);
+			const expectedCenter = latLng([0, 0]);
+			expect(map.flyToBounds(bounds)).to.equal(map);
+			expect(map.getCenter()).to.be.nearLatLng(expectedCenter);
+		});
+
+		it('flies to requested bounds and corresponding center (low zoom case)', function (done) {
+			this.timeout(10000);
```
Comment (Falke-Design) [inline spec/suites/map/MapSpec.js:2658]:
```suggestion
```

The tests should not need the 10 seconds. Please try to disable animation: `map.setView([0, 0], 0, {animate: false});`, `map.flyToBounds(bounds, {animate: false})`

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Please revert all the code changes from the auto formatter. 

After a quick look, the new tests are looking pretty good besides the timeout

**Comment by shreya024** [comment]
@Falke-Design Kindly review again

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@Falke-Design shall we merge this one in? Looks like a solid addition to the test suite.

**Review by Falke-Design — APPROVED** [review]

**Comment by jonkoops** [comment]
Thanks for your contribution @shreya024, it's much appreciated!

**Pull request merged**